domain/schema/model/sql/0022-resource.sql

```diff
@@ -50,15 +45,32 @@ CREATE TABLE resource (
     CONSTRAINT fk_resource_state_id
     FOREIGN KEY (state_id)
     REFERENCES resource_state (id),
-    CONSTRAINT fk_resource_hash_type_id
-    FOREIGN KEY (hash_type_id)
-    REFERENCES object_store_metadata_hash_type (id)
+    CONSTRAINT fk_resource_store_uuid
+    FOREIGN KEY (resource_store_uuid)
+    REFERENCES resource_store (uuid),
+    CONSTRAINT fk_resource_supplied_by_uuid
+    FOREIGN KEY (supplied_by_id)
+    REFERENCES resource_supplied_by (uuid)
 );
 
-CREATE UNIQUE INDEX idx_resource ON resource (application_uuid, name, state_id);
+-- Links applications to their resources
+CREATE TABLE application_resource (
+    application_uuid TEXT NOT NULL,
```

Comment:
**todo:** This table doesn't have any Primary key nor indexes.

As  far i checked, a resource is linked to one application,  so we should have a PRIMARY KEY on `resource_uuid`

**suggestion:** It may worth considering to add a simple INDEX on `application_uuid`, since we will query this stable through this field (to get all resources from a specific application)